lib/qwerty/quran_simple.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'text'
+require_relative 'config'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.